src/components/searchbar/searchbar.ts

```diff
@@ -116,6 +117,11 @@ export class Searchbar extends BaseInput<string> {
   }
 
   /**
+   * @input {string} Set the input's autocapitalize property. Values: `"none"`, `"sentences"`, `"words"`, `"characters"`. Default `"sentences"`.
+   */
+  @Input() autocapitalize: string = 'sentences';
```

Comment:
@kelvindart why sentences by default?

Reply:
This is the default behaviour, so I didn't want to introduce a breaking change (see: https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocapitalize).

> If the autocapitalize attribute is present but no value is specified (as in `<input autocapitalize>`), the default value is sentences. If the autocapitalize attribute is not present, the default value is sentences for form and none for input elements with type set to password as in `<input type="password">`. For all other form controls, the default value is inherited from the enclosing form element.